string[] bin_list       # List of bins to look at
---
MotionPlan[] motion_plan_list # List of motion plans to execute to look at bins
